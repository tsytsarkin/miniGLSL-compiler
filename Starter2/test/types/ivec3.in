{
    ivec3 i;
    ivec3 j = ivec3(1, 2, 3);
    j[2] = 5;
}
