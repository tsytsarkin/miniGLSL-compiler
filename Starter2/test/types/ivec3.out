  1: {
  2:     ivec3 i;
  3:     ivec3 j = ivec3(1, 2, 3);
  4:     j[2] = 5;
  5: }
empty declarations
IVEC3_T type
declaration (i)
declarations
IVEC3_T type
IVEC3_T type
INT_C (1) expression
INT_C (2) expression
INT_C (3) expression
constructor
constructor expression
assignment declaration (j)
declarations
empty statements
vector variable (j) [2]
INT_C (5) expression
assignment statement
statements
scope
program
